bears/js/HappinessLintBear.py

```diff
@@ -19,6 +19,19 @@ class HappinessLintBear:
     ASCIINEMA_URL = 'https://asciinema.org/a/80714'
     CAN_DETECT = {'Syntax'}
 
+    def __init__(self, use_spaces, queue):
```

Comment:
Lines 22-24 are redundant, they are already handled by the `@linter` decorator :)

Reply:
ooh is it, can u plz brief me what @linter does n how?

Reply:
yeah its mostly metaclass magic, hard to explain in a comment. Try reading the source and see if you can grok it